include/dslash.h

```diff
@@ -79,10 +77,10 @@ namespace quda
       strncat(aux[kernel_type], aux_base, TuneKey::aux_n - 1);
     }
 
-    virtual bool tuneGridDim() const { return arg.kernel_type == EXTERIOR_KERNEL_ALL && arg.shmem > 0; }
-    virtual unsigned int minThreads() const { return arg.threads; }
+    virtual bool tuneGridDim() const override { return arg.kernel_type == EXTERIOR_KERNEL_ALL && arg.shmem > 0; }
```

Comment:
In general, adding `override` is just a good habit to get into, and gives extra compile-time checking (from https://en.cppreference.com/w/cpp/language/override)
> In a member function declaration or definition, override specifier ensures that the function is virtual and is overriding a virtual function from a base class. The program is ill-formed (a compile-time error is generated) if this is not true.
>
>override is an identifier with a special meaning when used after member function declarators: it's not a reserved keyword otherwise.